src/shared/types/props/epic.ts

```diff
@@ -89,8 +90,9 @@ export const variantSchema = z.object({
     showSignInLink: z.boolean().optional(),
     bylineWithImage: bylineWithImageSchema.optional(),
     showChoiceCards: z.boolean().optional(),
-    choiceCardAmounts: selectedAmountsVariantSchema.optional(),
-    defaultChoiceCardFrequency: contributionFrequencySchema.optional(),
+    choiceCardsSettings: choiceCardsSettings.nullish(),
```

Comment:
We'll need to make a change to DCR to use this field when rendering the epic choice cards